api/src/changeRequests/processChangeRequest.ts

```diff
@@ -28,6 +29,15 @@ export async function processChangeRequest(
         changeDoc.docType = changeRequest.doc.type;
         changeDoc.updatedTimeUtc = upsertResult.updatedTimeUtc;
         changeDoc.changes = upsertResult.changes;
+        changeDoc.changedByUser = userId;
+
+        if (upsertResult.changes.memberOf) {
+            changeDoc.memberOf = upsertResult.changes.memberOf;
+        }
+
+        if (upsertResult.changes.acl) {
+            changeDoc.acl = upsertResult.changes.acl;
+        }
```

Comment:
I think tests are missing for this behaviour.

Reply:
Added tests for this in processChangeRequest.spec.ts:
"can insert a change document into the database for a correctly submitted new document with group membership"
"can insert a change document into the database for a correctly submitted new group document"